charts/argo-rollouts/templates/controller/deployment.yaml

```
@@ -101,6 +104,9 @@ spec:
       {{- end }}
       securityContext:
         {{- toYaml .Values.podSecurityContext | nindent 8 }}
+      {{- if .Values.controller.terminationGracePeriodSeconds }}
+      terminationGracePeriodSeconds: {{ .Values.controller.terminationGracePeriodSeconds }}
+      {{- end }}
```

```suggestion
      {{- with .Values.controller.terminationGracePeriodSeconds }}
      terminationGracePeriodSeconds: {{ . }}
      {{- end }}
```
Like @yu-croco suggested, please use `with` instead of `if`.  It is DRYer.